torax/_src/transport_model/bohm_gyrobohm.py

```diff
@@ -54,10 +54,11 @@ def __init__(
 
   def _call_implementation(
       self,
+      transport_dynamic_runtime_params: DynamicRuntimeParams,
```

Comment:
I have not checked but am not convinced this will type check as it is being based a list of the base classes. Either check the type in combined/base class or at the top of call_implementation

Reply:
Yes that is what I was meaning, can you change the type of this as well to the base class not the inherited to avoid errors